core/pta/stats.c

```diff
@@ -36,7 +37,17 @@
  * uint32_t    Biggest byte size which allocation failed
  * uint32_t    Biggest byte size which allocation succeeded
  */
+
 #define STATS_CMD_TA_STATS		3
+/*
+ * STATS_CMD_SYSTEM_TIME - Get system times
+ *
+ * [out]    value[0].a        REE time as seen by OP-TEE in seconds
+ * [out]    value[0].b        REE time as seen by OP-TEE, milliseconds part
+ * [out]    value[1].a        TEE system time in seconds
+ * [out]    value[1].b        TEE system time, milliseconds part
+ */
+#define STATS_CMD_SYSTEM_TIME		4
```

Comment:
Since it's not only "system" time but also "REE" time, I suppose `STATS_CMD_GET_TIME` would be a better name

Reply:
good point, thanks.